cmd/gossamer/configcmd.go

```diff
@@ -83,19 +84,20 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 
 	log.Info("🕸\t Configuring node...", "datadir", fig.Global.DataDir, "protocolID", string(gendata.ProtocolId), "bootnodes", fig.P2p.BootstrapNodes)
 
-	// TODO: BABE
-	msgRec := make(chan p2p.Message)
+	bsChan := make(chan types.Block)
```

Comment:
Not sure what to name the channel before it becomes a send or receive channel...

Reply:
See comment below